example/demo/src/index.ts

```diff
@@ -1,9 +1,35 @@
-import { createWSServer } from '@rise-tools/server'
+import '@rise-tools/kit-react-navigation/server'
+
+import { createConnectionQR, getHost, HostType } from '@rise-tools/cli'
+import { createHTTPServer } from '@rise-tools/server'
 
 import { models as delivery } from './delivery/ui'
 import { models as inventory } from './inventory/ui'
 import { models as controls } from './ui-controls/ui'
 
-createWSServer({ ...inventory, ...controls, ...delivery }, Number(process.env.PORT || '3005'))
+const models = { ...inventory, ...controls, ...delivery }
+
+async function init() {
+  const port = Number(process.env.PORT || '3005')
+  const host = await getHost(HostType.tunnel)
+
+  // await createWSServer(models, port)
+  await createHTTPServer(models, port)
```

Comment:
Can we use createServer here?

Also, maybe the server should accept an argument for host

Reply:
I am surprised, but also excited, that `http` server works! Last time I tried it was broken! 😆